<launch>

	<!-- Participant ID -->
	<arg name="ID" default="0"/>

	<!-- Experimental task (FAM - familiarization, EXP - experimental)-->
	<arg name="task" default="EXP"/> 

	<!-- Method type ('A'- impedance, 'B'- learning, 'C'- demonstration) -->
	<arg name="methodType" default="A"/>

	<!-- Demo mode ('T' - demo with optimal weights, 'F' - default weights) -->
	<arg name="demo" default="F"/>

	<!-- Record data mode -->
	<arg name="record" default="F"/>

	<!-- Feature learning method (ALL, MAX, BETA)-->
	<arg name="featMethod" default="ALL"/>

	<!-- List of string features that need correcting -->
	<arg name="featList" default="table"/>

	<!-- Location with trajectory cache -->
	<arg name="traj_cache" default=None/>
	
    <!-- Location with trajectory random -->
	<arg name="traj_rand" default=None/>
	
    <!-- Location with trajectory optimal -->
	<arg name="traj_optimal" default=None/>
	
    <!-- velocity-based with trajopt planning PID -->
	<node name="pid_trajopt" pkg="iact_control" type="pid_trajopt.py"
    args="$(arg ID) $(arg task) $(arg methodType) $(arg demo) $(arg record)
    $(arg featMethod) $(arg featList) $(arg traj_cache) $(arg traj_rand) $(arg
    traj_optimal)" output="screen"/>

</launch>
